_includes/manuals/1.10/1.2_release_notes.md

```diff
@@ -2,6 +2,40 @@
 
 ### Headline features
 
+#### Orchestration rebuilder
```

Comment:
+1, I think Orchestration is fine, particularly when coupled with the first paragraph below which clearly explains the intention.